src/constants.js

```diff
@@ -0,0 +1,6 @@
+export const LayerNames = {
```

Comment:
Why the `L` in uppercase?

Reply:
Because it's a constant

Reply:
Is it not all characters in uppercase? Like:
```
const COLORS = [50, 100, 200, 300, 400, 500, 600, 700, 800, 900, 950];
```

Reply:
See together, enum has uppercase for its keys.